browser/brave_adaptive_captcha/sources.gni

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//brave/components/brave_adaptive_captcha/buildflags/buildflags.gni")
+
+brave_browser_brave_adaptive_captcha_sources = []
```

Comment:
Do not do this. You should be creating a new BUILD.gn file for this directory

Reply:
Fixed.

Reply:
I'm confused, is there a fix that isn't pushed? There should not be a sources.gni file here

Reply:
Sorry, I should have explained that I added this for the adaptive captcha factory to resolve a circular dependency issue.